### ISSUE TITLE
The 'Hide filters' and 'Remove all' do not need colour contrast requirements.

### ISSUE THREAD

**Issue by ivanteoh** [eea/eea.facetednavigation#94]
More info on colour contrast at https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html. The font colour of these texts are #95b229. It fail the requirement when verifying at http://webaim.org/resources/contrastchecker/

**Comment by avoinea** [comment]
Replaced with Plone 5 default colors.
